packages/neos-ui-sagas/src/index.js

```diff
@@ -0,0 +1,29 @@
+import * as browser from './Browser/index';
+import * as changes from './Changes/index';
+import * as crContentDimensions from './CR/ContentDimensions/index';
+import * as crNodeOperations from './CR/NodeOperations/index';
+import * as crPolicies from './CR/Policies/index';
+import * as publish from './Publish/index';
+import * as serverFeedback from './ServerFeedback/index';
+import * as uiContentCanvas from './UI/ContentCanvas/index';
+import * as uiContentTree from './UI/ContentTree/index';
+import * as uiEditPreviewMode from './UI/EditPreviewMode/index';
+import * as uiInspector from './UI/Inspector/index';
+import * as uiPageTree from './UI/PageTree/index';
+import * as uiHotkeys from './UI/Hotkeys/index';
+
+module.exports = {
+    browser,
+    changes,
+    crContentDimensions,
+    crNodeOperations,
+    crPolicies,
+    publish,
+    serverFeedback,
+    uiContentCanvas,
+    uiContentTree,
+    uiEditPreviewMode,
+    uiInspector,
+    uiPageTree,
+    uiHotkeys
```

Comment:
Hey Dmitri,

I somehow dislike that these names will then become "API" in some sense.

I do not yet understand why we need to do this; as one could do the following in a custom manifest:

```
watchPersist = sagasRegistry.get('neos-ui/Changes/watchPersist');
sagasRegistry.set('neos-ui/Changes/watchPersist', overrideWatchPersist(watchPersist);
```

This way the only API is the registry keys in the SagasRegistry. What do you think? :-) Then it does not matter at all if the sagas reside in a custom package or not.

Reply:
Oh, I didn't think about it, a very fair point.
However I'd still like to move Sagas out into a separate package, just not expose it via extensibility API, for the sake of consistency with other packages and just to structure the code a bit more clearly (e.g. highlight decoupled nature of sagas).